src/ipyradiant/rdf2nx/literal_converter.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020 ipyradiant contributors.
+# Distributed under the terms of the Modified BSD License.
+import logging
+from datetime import date
+from typing import Any, Callable, Dict
+
+from rdflib import Literal, URIRef
+from rdflib.namespace import XSD
+
+logger = logging.getLogger(__name__)
+
+
+# TODO #63
+XSD2PY: Dict[URIRef, Callable] = {
```

Comment:
If @dfreeman06 @nrbgt have any opinions on these mappings from XSD to py I'd appreciate it. The commented ones still have no mapping.

Reply:
dateTime is particularly tough since XSD dateTime supports some crazy syntax for things like date ranges.